assets/home/sections/get-help.js

```diff
@@ -25,34 +27,33 @@ import Section from '../section';
 const Item = ( { title, url, extraLink } ) => {
 	const isTitleInteractive = url !== null;
 
-	const link = isTitleInteractive ? (
-		<a href={ url } target="_blank" rel="noreferrer">
-			{ title }
-		</a>
-	) : (
-		title
-	);
-
 	return (
 		<li className="sensei-home__help-item">
 			<div className="sensei-home__help-item__icon">
-				<Icon icon={ isTitleInteractive ? help : lock } size={ 24 } />
+				{ isTitleInteractive ? (
+					<Icon icon={ help } />
+				) : (
+					<LockIcon
+						className={ 'sensei-home__help-item__icon__lock' }
+					/>
+				) }
 			</div>
 			<div
 				className={ classNames( 'sensei-home__help-item__title', {
 					'sensei-home__help-item__title--disabled': ! isTitleInteractive,
 				} ) }
 			>
-				{ link }
-			</div>
-			{ extraLink && (
-				<div className="sensei-home__help-item__extra-link">
-					<a href={ extraLink.url } target="_blank" rel="noreferrer">
-						{ extraLink.label }{ ' ' }
-						<Icon icon={ external } size={ 16 } />
+				{ isTitleInteractive ? (
+					<a href={ url } target="_blank" rel="noreferrer">
```

Comment:
Should we use `<Link>` here, too since all of these links are also external?

Reply:
I thought the same. I have asked Andrei and depending on his answer I will open a ticket to adapt to that or not :)